source/scenes/common/Import/ImportPhrase/ImportPhrase.container.tsx

```diff
@@ -62,32 +66,61 @@ const ImportPhraseContainer: FC<IImportPhraseContainer> = ({
   };
 
   const onSubmit = async (phraseArray: string[]) => {
-    setIsDisabled(true);
+    setIsLoading(true);
     const phrase = phraseArray.join(' ');
 
     if (type === IMPORT_TYPE.RESTORE) {
       if (walletController.onboardHelper.importAndValidateSeedPhrase(phrase)) {
         onButtonPress();
```

Comment:
I am not super sure if `onButtonPress` would be the correct naming for this, maybe something more explicit, I see no button being exactly pressed in the component, maybe something like `onImportPhraseSuccess`, something more explicit.